shelley/chain-and-ledger/shelley-spec-ledger-test/src/Test/Shelley/Spec/Ledger/Shrinkers.hs

```diff
@@ -40,16 +42,22 @@ shrinkTx (Tx _b _ws _md) =
   [Tx b' _ws _md | b' <- shrinkTxBody _b]
 
 shrinkTxBody ::
```

Comment:
Let's document the weirdness here; this is a shrinker for the _Shelley_ `TxBody`, but has code in it to deal with `Value`s, which will only ever exist in the _ShelleyMA_ TxBody (which is different, and will have its own shrinker).

This is fine for now, but let's add a comment suggesting the better resolution: when we have a shrinker for the ShelleyMA TxBody, let's use this logic to allow shrinking the values, and revert this function to its prior incarnation, fixed to `Coin`